aptos-move/e2e-benchmark/src/main.rs

```diff
@@ -247,33 +252,44 @@ fn main() {
                 100
             },
         );
-        let diff = (elapsed_micros as f32 - expected_time) / expected_time * 100.0;
+        let diff = (measurement.elapsed_micros as f32 - expected_time as f32)
+            / (expected_time as f32)
+            * 100.0;
         println!(
-            "{:15}  {:15.1}  {:14.1}%   {:?}",
-            elapsed_micros, expected_time, diff, entry_point
+            "{:13} {:13.1} {:12.1}% {:13} {:13} {:13}  {:?}",
+            measurement.elapsed_micros,
+            expected_time,
+            diff,
+            (measurement.execution_gas + measurement.io_gas) as u128 / measurement.elapsed_micros,
+            measurement.execution_gas,
+            measurement.io_gas,
+            entry_point
         );
 
         json_lines.push(json!({
             "grep": "grep_json_aptos_move_vm_perf",
             "transaction_type": entry_point_name,
-            "wall_time_us": elapsed_micros,
+            "wall_time_us": measurement.elapsed_micros,
+            "gps": (measurement.execution_gas + measurement.io_gas) as u128 / measurement.elapsed_micros,
```

Comment:
`gas/us` for better readability? Plus it is not seconds?

Reply:
parsing with / in logs can be more complicated - this json is for humio graphs

gas unit is 1/10^6 gas internal units, and same as micros, so this is gas unit / s

Reply:
I see, can you maybe put a comment for that here? gps is ok 👍